main/py-flake8/APKBUILD

```diff
@@ -56,4 +57,15 @@ _py() {
 	done
 }
 
+_getpythonver(){
+	echo $($1 -c 'import sys; print("%i%i" % (sys.version_info.major, sys.version_info.minor))')
+}
+
+check() {
+	cd "$builddir"
+	python2 setup.py check
+	python3 setup.py check
+	tox -e py$(_getpythonver python2),py$(_getpythonver python3)
```

Comment:
instead of tox probably better to move required packages from testing?
btw new release is out https://gitlab.com/pycqa/flake8/commit/b6bd3a8ddd88dd29a94b5fa8a6634e5dffe7a711